src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -36,19 +36,28 @@
 
   ([sqs-client name
     {:keys [fifo
+            visibility-timeout-in-seconds
             kms-master-key-id
             kms-data-key-reuse-period]}]
    (let [request (cond-> (CreateQueueRequest. name)
                    fifo
                    (doto (.addAttributesEntry
-                           "FifoQueue" "true"))
+                           "FifoQueue"
+                           "true"))
+
+                   visibility-timeout-in-seconds
+                   (doto (.addAttributesEntry
+                           "VisibilityTimeout"
+                           (str visibility-timeout-in-seconds)))
+
                    kms-master-key-id
-                   (.addAttributesEntry
-                     "KmsMasterKeyId" kms-master-key-id)
+                   (doto (.addAttributesEntry
+                           "KmsMasterKeyId" kms-master-key-id))
 
                    kms-data-key-reuse-period
-                   (.addAttributesEntry
-                     "KmsDataKeyReusePeriodSeconds" kms-data-key-reuse-period))]
+                   (doto (.addAttributesEntry
```

Comment:
this `cond->` makes this weird if more than one of these keys are not-nil. e.g. if `kms-master-key-id` and `kms-data-key-reuse-period` are truthy, we'd end up with equivalent of,

```
(-> (CreateQueueRequest. name)
   (doto (.addAttributesEntry "KmsMasterKeyId" kms-master-key-id))
  (doto (.addAttributesEntry "KmsDataKeyReusePeriodSeconds" kms-data-key-reuse-period))
```

so the result of 2nd line would be passed as the first arg of the last `doto`.

Reply:
Perhaps refactor out this `cond->` block into a fn and perform a unit test to ensure that the object is created as expected when we have multiple of these keys set.

Reply:
You're right, I'll try to reshape it.